src/meson.build

```diff
@@ -19,6 +19,13 @@ gdbus_src = custom_target(
 
 daemon_builddir = meson.current_build_dir()
 
+unitdir = ''
+systemd = dependency('systemd')
+unitdir = systemd.get_pkgconfig_variable('systemdsystemunitdir')
+
+confsystemd = configuration_data()
```

Comment:
I'd move this to just above `systemd_unit = configure_file(`

Reply:
right, makes sense, I moved it to one section